go_lib/dependency/helm/helm.go

```diff
@@ -213,3 +227,21 @@ func getActionConfig(namespace string) (*action.Configuration, error) {
 	}
 	return actionConfig, nil
 }
+
+func getMD5Hash(templates, values map[string]interface{}) string {
```

Comment:
well, on the large maps it will always generate a new hash. Probably we need something like this:
```
keys := make([]string, len(m))
	i := 0
	for k := range values {
		keys[i] = k
		i++
	}

    sort.Strings(keys)
    for _, k := range keys {
		v := values[k]
		b := sha256.Sum256([]byte(fmt.Sprintf("%v", k)))
		h.Write(b[:])
		b = sha256.Sum256([]byte(fmt.Sprintf("%v", v)))
		h.Write(b[:])
	}
```